actions/image_partition_action.go

```diff
@@ -642,12 +643,14 @@ func (i *ImagePartitionAction) Verify(context *debos.DebosContext) error {
 		}
 
 		if p.Start == "" {
-			return fmt.Errorf("Partition %s missing start", p.Name)
+			p.Start = prevEnd
```

Comment:
It'd be good to move this logic into the `Run` stage rather than `Verify` stage potentially with some comment  about what is going on.